css-borders-4/Overview.bs

```diff
@@ -741,25 +751,19 @@ Initial: none
 Applies to: all elements
 Inherited: no
 Percentages: N/A
-Computed value: either the keyword ''box-shadow-offset/none'' or
-    a list, each item consisting of four absolute lengths
-    plus a computed color and optionally also a ''box-shadow-position/inset'' keyword
-Animation type: by computed value,
-    treating ''box-shadow-offset/none'' as a zero-item list
-    and appending blank shadows (''transparent 0 0 0 0'')
-    with a corresponding ''box-shadow-position/inset'' keyword as needed
-    to match the longer list
-    if the shorter list is otherwise compatible with the longer one
+Computed value: see individual properties
+Animation type: see individual properties
 </pre>
 
   <p>The 'box-shadow' property attaches one or more drop-shadows to the box.
-  The property accepts either the ''box-shadow-offset/none'' value, which indicates no shadows,
-  or a comma-separated list of shadows, ordered front to back.
+  The property accepts a comma-separated list of shadows,
+  ordered front to back.
 
   <p>Each shadow is given as a <<spread-shadow>>,
   outlining the 'box-shadow-offset', and optional values for the 'box-shadow-blur',
   'box-shadow-spread', 'box-shadow-color', and 'box-shadow-position'.
-  Omitted lengths are 0; omitted colors default to the ''currentcolor'' value.
+  If the offset is ''box-shadow-offset/none'',
+  ''box-shadow-color'' is treated as ''transparent''.
```

Comment:
"Treated as" sounds like changing the used value of `box-shadow-color`, which can be a possible approach, but the shorthand wouldn't be the correct place for it, and it would need more complication than that.

What I was thinking (maybe it's not the best behavior, but it's simple to specify for now, better approaches can be considered in a follow-up) is that the specified value of `box-shadow-color` would be `transparent`, so no "treating as" necessary.

Don't remove the default length either.

```suggestion
  Omitted lengths are 0;
  omitted colors default to the ''transparent'' value when the specified offset is ''none''
  and to ''currentcolor'' otherwise.
```

Reply:
Pushed this change with a slight adjustment.